### PR TITLE
chore(cxl-ui): upgrade @vaadin/web-components to v.21.0.0-alpha-3

### PR DESCRIPTION
**Pull request by anoblet**

### CODE REVIEW

**Comment by anoblet** [comment]
Upgrade seems to break `CXL Lumo Styles/Elements`.

**Comment by anoblet** [comment]
Also introduces an endless build:


https://user-images.githubusercontent.com/7674171/120508484-c524d180-c395-11eb-88b6-6a5c0298d88b.mp4

**Comment by lkraav** [comment]
> Upgrade breaks `CXL Lumo Styles/Elements`

Salvageable?

**Comment by anoblet** [comment]
I believe so. This pops up in console:

```
vendors~main.3f3ecec1.iframe.bundle.js:2 Unexpected error while loading ./cxl-lumo-styles/elements.stories.js: NotSupportedError: Failed to execute 'define' on 'CustomElementRegistry': the name "vaadin-lumo-styles" has already been used with this registry
```

Hopefully it's a user-land issue.

**Comment by lkraav** [comment]
> Failed to execute 'define' on 'CustomElementRegistry': the name "vaadin-lumo-styles" has already been used with this registry

Historically, this has been about multiple packages copies in `node_modules/` :thinking:

File: packages/cxl-ui/package.json [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
-    "@vaadin/vaadin-notification": "^2.0.0-alpha1",
-    "@vaadin/vaadin-progress-bar": "^2.0.0-alpha2",
-    "@vaadin/vaadin-tabs": "^4.0.0-alpha4",
+    "@vaadin/vaadin-accordion": "21.0.0-alpha3",
+    "@vaadin/vaadin-button": "21.0.0-alpha3",
+    "@vaadin/vaadin-context-menu": "21.0.0-alpha3",
+    "@vaadin/vaadin-notification": "21.0.0-alpha3",
+    "@vaadin/vaadin-progress-bar": "21.0.0-alpha3",
+    "@vaadin/vaadin-tabs": "21.0.0-alpha3",
     "@vaadin/vaadin-themable-mixin": "^1.6.2",
```
Comment (lkraav) [inline packages/cxl-ui/package.json:24]: But also, why hasn't `@vaadin/vaadin-themable-mixin` been bumped?

https://github.com/vaadin/web-components/tree/master/packages/vaadin-themable-mixin should be brought along for the ride, no?

**Comment by anoblet** [comment]
> > Failed to execute 'define' on 'CustomElementRegistry': the name "vaadin-lumo-styles" has already been used with this registry
> 
> Historically, this has been about multiple packages copies in `node_modules/` 🤔

The amount of times I've run `git clean -fxd && yarn install && yarn build`...

I bumped > > Failed to execute 'define' on 'CustomElementRegistry': the name "vaadin-lumo-styles" has already been used with this registry
> 
> Historically, this has been about multiple packages copies in `node_modules/` 🤔

I've run `git clean -fxd && yarn install && yarn build` more times than I can count.

I bumped `@vaadin/vaadin-themable-mixin` still the same issue.

File: packages/cxl-ui/package.json (hunk 1 above)
Comment (anoblet) [reply packages/cxl-ui/package.json]: Agreed

**Comment by anoblet** [comment]
New PR: https://github.com/conversionxl/aybolit/pull/184

**Pull request closed without merging**